auth-password-store.el

```diff
@@ -45,10 +45,18 @@
 See `auth-source-search' for details on SPEC."
   (cl-assert (or (null type) (eq type (oref backend type)))
              t "Invalid password-store search: %s %s")
-  (when (listp host)
+  (when (and host (listp host))
```

Comment:
I think that is the same as `(consp host)`. You should be able to verify that with the unit tests :-).

Reply:
It seems to be, but what do you mean RE: being able to verify with the unit tests?